src/lib/_imports/components/c-card/c-card.demo.images.css

```diff
@@ -19,18 +19,3 @@
 section > section {
     margin: unset;
 }
-
-/* Contact card adjustments */
-:is(.card, .c-card, [class*="card--"]) h4 {
-    margin-bottom: 0.375em;
-}
-
-:is(.card, .c-card, [class*="card--"]) p {
-    margin-top: 0px;
-    margin-bottom: 10px;
-}
-
-.card--image-top {
-    max-width: fit-content;
-}
-
```

Comment:
I think this style has no effect. If it has effect, please correct me.

Reply:
Hmm, I think I may have used a wrong style here. I think what I was trying to account for was this.

I tested `width: fit-content`, but I don't think that is the whole issue here... 

<img width="596" alt="Screenshot 2024-01-09 at 10 55 33 AM" src="https://github.com/TACC/Core-Styles/assets/63771558/8a5f3261-d516-48e0-bbaf-6e183761cf99">

Reply:
You did. You are correct. I know how to make it work, and fix "the whole issue". Please review the isolated solution in:
- #296

Reply:
And, a tiny fix for this PR: 3a5bd17.

<img width="960" alt="3a5bd17" src="https://github.com/TACC/Core-Styles/assets/62723358/4fae21fa-1968-4501-b1e6-8a81e51f356d">